CMakeLists.txt

```diff
@@ -270,6 +270,22 @@ if(HERMES_RPC_THALLIUM)
   endif()
 endif()
 
+# GLOG
+find_package(glog REQUIRED)
+if(glog_FOUND)
+  message(STATUS "found GLOG at ${glog_DIR}")
+endif()
+find_path(GLOG_INCLUDE_DIRS NAME glog PATH_SUFFIXES include/)
+include_directories(${GLOG_INCLUDE_DIRS})
+
+# GFLAGS
+find_package(gflags REQUIRED)
```

Comment:
I would prefer to make gflags an optional dependency, or remove it altogether. I don't use it for anything, and glog works fine without it.

Reply:
I think glog depends on gflags.
```
/home/runner/work/hermes/hermes/spack/opt/spack/linux-ubuntu20.04-x86_64_v4/gcc-10.3.0/glog-0.4.0-jcohf3eh7cax7d6742d6owabbwdj6464/include/glog/logging.h:86:10: fatal error: gflags/gflags.h: No such file or directory
   86 | #include <gflags/gflags.h>
      |          ^~~~~~~~~~~~~~~~~
```

Reply:
You can disable the gflags dependency [via cmake](https://github.com/google/glog/blob/master/CMakeLists.txt#L41). Unfortunately it looks like the [glog spack package](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/glog/package.py) doesn't expose a variant without gflags. It hard-codes the dependency. I guess we're stuck with it for now.